src/endpoints/speech.js

```diff
@@ -43,8 +43,8 @@ router.post('/recognize', jsonParser, async (req, res) => {
         const start = performance.now();
         const result = await pipe(wav, { language: lang || null, task: 'transcribe' });
         const end = performance.now();
-        console.log(`Execution duration: ${(end - start) / 1000} seconds`);
-        console.log('Transcribed audio:', result.text);
+        console.debug(`Execution duration: ${(end - start) / 1000} seconds`);
+        console.debug('Transcribed audio:', result.text);
```

Comment:
I feel like one line in the speech endpoint should be an info, not just debug, to show that speech gen happened.